fallback/pom.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Not related to this file, but I noticed that README.md is missing. It has a very specific format described here: https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute You can also check the other patterns for examples.